static/app/components/profiling/flamegraph/flamegraphDrawer/differentialFlamegraphDrawer.tsx

```diff
@@ -386,17 +390,16 @@ const ProfilingDetailsListItem = styled('li')<{
   }
 `;
 
-const StyledButton = styled(Button)<{active: boolean}>`
-  border: none;
-  background-color: transparent;
-  box-shadow: none;
-  transition: none !important;
+const StyledButton = styled('button')<{active: boolean}>`
```

Comment:
I don’t think switching to `’button’` is a good idea. For example, I don’t think focus-styles will work here unless we re-add them?

We have a Button variant that is flat (non-chonk) - can’t we use that one?

Reply:
There is one annoying thing with the transparent variant, which is that it only exists in chonk, meaning we need to @ts-ignore or expect the error if we want to use it. I think we could do that, as @ts-expect error will throw once chonk is released, so the cleanup would be somewhat easier to manage. I can try doing that and seeing what the ergonomics of that would be